src/release/publisher.ts

```diff
@@ -753,4 +757,11 @@ export interface GitPublishOptions extends VersionArtifactOptions {
    * @default "main"
    */
   readonly gitBranch?: string;
+  
+  /**
+   * Override git-push command. 
+   * 
+   * Set to null to disable pushing.
+   */
+  readonly gitPushCommand?: string | null;
```

Comment:
```suggestion
  readonly gitPushCommand?: string;
```

Projen is compiled with JSII to other languages where `undefined` and `null` will be treated as equivalent, so distinguishing between them may lead to errors.

How about we instead add a line in the docstring that says e.g. `To disable pushing, use a no-op command such as "true"`?

Reply:
Thanks @Chriscbr. How about, `To disable pushing, set to an empty string`? 

I feel `gitPushCommand: true` reads a little weirdly for something which is disabling something.

Reply:
Fair point. I suppose the alternative here is to add another field. This solution sounds good to me. 🙂